xbmc/pvr/channels/PVRChannel.h

```diff
@@ -416,6 +416,18 @@ namespace PVR
 
     static std::string GetEncryptionName(int iCaid);
 
+    /*!
+     * @brief Get the client order for this channel
+     * @return iOrder The order for this channel
+     */
+    int ClientOrder(void) const { return m_iOrder; }
```

Comment:
(void) -> ()

Reply:
Ah, ok. Was sticking with the what was used in the rest of file.

Reply:
For new code we apply current coding guidelines, regardless of the format of the existing code.